tests/components/homewizard/test_number.py

```diff
@@ -45,8 +46,13 @@ async def test_number_entities(
     # Test unknown handling
     assert state.state == "100"
 
+    # Construct empty response
+    device = Device.from_dict(
+        load_json_object_fixture(f"{device_fixture}/device.json", DOMAIN)
+    )
+
     mock_homewizardenergy.combined.return_value = CombinedModels(
-        device=None, measurement=Measurement(), system=System(), state=State()
+        device=device, measurement=Measurement(), system=System(), state=State()
```

Comment:
`device=none` is 'illigal', as this value is non-optional. Having this none breaks a check in python-homewizard-energy